From ee713313d7e14097626e72e6c0241bebc9210e80 Mon Sep 17 00:00:00 2001
From: "Steven Rostedt (Red Hat)" <rostedt@goodmis.org>
Date: Tue, 22 Nov 2016 11:26:14 -0500
Subject: [PATCH 121/134] trace-cmd: Make the parse_*() functions
 tracecmd_parse_*()

To allow for better namespace of the tracecmd library, rename the parse_*()
functions to tracecmd_parse_*() to prevent namespace collisions.

Signed-off-by: Steven Rostedt <rostedt@goodmis.org>
---
 trace-cmd.h   |  8 ++++----
 trace-input.c | 10 +++++-----
 trace-read.c  |  2 +-
 trace-util.c  | 12 ++++++------
 4 files changed, 16 insertions(+), 16 deletions(-)

diff --git a/trace-cmd.h b/trace-cmd.h
index 6a229100b84b..485907f2b7f0 100644
--- a/trace-cmd.h
+++ b/trace-cmd.h
@@ -27,10 +27,10 @@
 #define TRACECMD_ERROR(ret)	((void *)((unsigned long)(ret) | TRACECMD_ERR_MSK))
 #define TRACECMD_PTR2ERR(ptr)	((unisgned long)(ptr) & ~TRACECMD_ERR_MSK)
 
-void parse_cmdlines(struct pevent *pevent, char *file, int size);
-void parse_trace_clock(struct pevent *pevent, char *file, int size);
-void parse_proc_kallsyms(struct pevent *pevent, char *file, unsigned int size);
-void parse_ftrace_printk(struct pevent *pevent, char *file, unsigned int size);
+void tracecmd_parse_cmdlines(struct pevent *pevent, char *file, int size);
+void tracecmd_parse_trace_clock(struct pevent *pevent, char *file, int size);
+void tracecmd_parse_proc_kallsyms(struct pevent *pevent, char *file, unsigned int size);
+void tracecmd_parse_ftrace_printk(struct pevent *pevent, char *file, unsigned int size);
 
 extern int tracecmd_disable_sys_plugins;
 extern int tracecmd_disable_plugins;
diff --git a/trace-input.c b/trace-input.c
index b90b31e4ec67..e676c8550714 100644
--- a/trace-input.c
+++ b/trace-input.c
@@ -694,7 +694,7 @@ static int read_proc_kallsyms(struct tracecmd_input *handle)
 	}
 	buf[size] = 0;
 
-	parse_proc_kallsyms(pevent, buf, size);
+	tracecmd_parse_proc_kallsyms(pevent, buf, size);
 
 	free(buf);
 	return 0;
@@ -722,7 +722,7 @@ static int read_ftrace_printk(struct tracecmd_input *handle)
 
 	buf[size] = 0;
 
-	parse_ftrace_printk(handle->pevent, buf, size);
+	tracecmd_parse_ftrace_printk(handle->pevent, buf, size);
 
 	free(buf);
 
@@ -2338,7 +2338,7 @@ static int read_and_parse_cmdlines(struct tracecmd_input *handle)
 	if (read_data_and_size(handle, &cmdlines, &size) < 0)
 		return -1;
 	cmdlines[size] = 0;
-	parse_cmdlines(pevent, cmdlines, size);
+	tracecmd_parse_cmdlines(pevent, cmdlines, size);
 	free(cmdlines);
 	return 0;
 }
@@ -2352,7 +2352,7 @@ static int read_and_parse_trace_clock(struct tracecmd_input *handle,
 	if (read_data_and_size(handle, &trace_clock, &size) < 0)
 		return -1;
 	trace_clock[size] = 0;
-	parse_trace_clock(pevent, trace_clock, size);
+	tracecmd_parse_trace_clock(pevent, trace_clock, size);
 	free(trace_clock);
 	return 0;
 }
@@ -2389,7 +2389,7 @@ int tracecmd_init_data(struct tracecmd_input *handle)
 		if (read_and_parse_trace_clock(handle, pevent) < 0) {
 			char clock[] = "[local]";
 			warning("File has trace_clock bug, using local clock");
-			parse_trace_clock(pevent, clock, 8);
+			tracecmd_parse_trace_clock(pevent, clock, 8);
 		}
 	}
 
diff --git a/trace-read.c b/trace-read.c
index 685a81bef8b6..48a603ca0d2a 100644
--- a/trace-read.c
+++ b/trace-read.c
@@ -1322,7 +1322,7 @@ static void add_functions(struct pevent *pevent, const char *file)
 		die("Failed to allocate for function buffer");
 	read_file_fd(fd, buf, st.st_size);
 	close(fd);
-	parse_proc_kallsyms(pevent, buf, st.st_size);
+	tracecmd_parse_proc_kallsyms(pevent, buf, st.st_size);
 	free(buf);
 }
 
diff --git a/trace-util.c b/trace-util.c
index cb8ada158f92..4073c790d5ae 100644
--- a/trace-util.c
+++ b/trace-util.c
@@ -379,8 +379,8 @@ void trace_util_print_plugin_options(struct trace_seq *s)
 	}
 }
 
-void parse_cmdlines(struct pevent *pevent,
-		    char *file, int size __maybe_unused)
+void tracecmd_parse_cmdlines(struct pevent *pevent,
+			     char *file, int size __maybe_unused)
 {
 	char *comm;
 	char *line;
@@ -408,8 +408,8 @@ static void extract_trace_clock(struct pevent *pevent, char *line)
 	free(clock);
 }
 
-void parse_trace_clock(struct pevent *pevent,
-			char *file, int size __maybe_unused)
+void tracecmd_parse_trace_clock(struct pevent *pevent,
+				char *file, int size __maybe_unused)
 {
 	char *line;
 	char *next = NULL;
@@ -423,7 +423,7 @@ void parse_trace_clock(struct pevent *pevent,
 	}
 }
 
-void parse_proc_kallsyms(struct pevent *pevent,
+void tracecmd_parse_proc_kallsyms(struct pevent *pevent,
 			 char *file, unsigned int size __maybe_unused)
 {
 	unsigned long long addr;
@@ -468,7 +468,7 @@ void parse_proc_kallsyms(struct pevent *pevent,
 	}
 }
 
-void parse_ftrace_printk(struct pevent *pevent,
+void tracecmd_parse_ftrace_printk(struct pevent *pevent,
 			 char *file, unsigned int size __maybe_unused)
 {
 	unsigned long long addr;
-- 
2.5.5

